### PR TITLE
Added get_tenant_info-command

### PR DESCRIPTION
**Pull request by yilmazdincer**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A command has been added to the integration. With this command we can see tenant information. This command provides information such as Endpoint expiration time.

## Screenshots
image
abc

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by adi88d** [comment]
Hi @yilmazdincer Thank you for your contribution!
please register your contribution by filling the contribution registration form in - https://forms.gle/XDfxU4E61ZwEESSMA

**Review by adi88d — CHANGES_REQUESTED** [review]
Please see my comments below:
- You added the command `xdr-get-tenant-info` to the `CortexXDRIR.py` file only, please update the `CortexXDRIR.yml` too with this command.
- After update the yml file you can update the `README.md` file with the new command - use `demisto-sdk generate-docs` command - for more information please see: https://xsoar.pan.dev/docs/concepts/demisto-sdk#generate-docs.
- Update `command_examples` file.
- Please bump the version of the pack in the pack_metadata.json file and add notes describing your changes to a new Release Notes file. 
To do that, please run the `demisto-sdk update-release-notes` command - for more information please see: https://xsoar.pan.dev/docs/concepts/demisto-sdk#update-release-notes.

File: Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py [hunk 1]
```diff
@@ -375,7 +375,20 @@ def replace_featured_field(self, field_type: str, fields: list[dict]) -> dict:
 
         return reply.get('reply')
 
+    def get_tenant_info(self):
+        request_data: Dict[str, Any] = {}
+        reply = self._http_request(
+            method='POST',
+            url_suffix='/system/get_tenant_info/',
+            json_data={'request_data': request_data},
```
Comment (adi88d) [inline Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py:383]: no need to create request_data if its empty dict, remove it and change line 383 to:

```
            json_data={'request_data': request_data},
```

**Comment by yilmazdincer** [comment]
> Please see my comments below:
> 
> * You added the command `xdr-get-tenant-info` to the `CortexXDRIR.py` file only, please update the `CortexXDRIR.yml` too with this command.
> * After update the yml file you can update the `README.md` file with the new command - use `demisto-sdk generate-docs` command - for more information please see: https://xsoar.pan.dev/docs/concepts/demisto-sdk#generate-docs.
> * Update `command_examples` file.
> * Please bump the version of the pack in the pack_metadata.json file and add notes describing your changes to a new Release Notes file.
>   To do that, please run the `demisto-sdk update-release-notes` command - for more information please see: https://xsoar.pan.dev/docs/concepts/demisto-sdk#update-release-notes.

**Review by adi88d — CHANGES_REQUESTED** [review]
Great Job!
- Please add outputs to the `xdr-get-tenant-info` command, you can use [generate-outputs](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/generate_outputs/README.md) from the SDK.
- Update the README.md file only with `xdr-get-tenant-info` command part.
- Remove the Pipfile

File: Packs/CortexXDR/ReleaseNotes/4_9_15.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Palo Alto Networks Cortex XDR - Investigation and Response
+- Added a new command 'xdr-get-tenant-info'
```
Comment (adi88d) [inline Packs/CortexXDR/ReleaseNotes/4_9_15.md:4]:
```suggestion
- Added the ***xdr-get-tenant-info*** command.
- Updated the Docker image to: *demisto/python3:3.10.9.42476*.
```

File: Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py [hunk 3]
```diff
@@ -375,6 +375,20 @@ def replace_featured_field(self, field_type: str, fields: list[dict]) -> dict:
...
+            url_suffix='/system/get_tenant_info/',
+            json_data={'request_data': {}},
+            timeout=self.timeout
+        )
+        return reply.get('reply', {})
+
+
+def get_tenant_info_command(client: Client):
+    tenant_info = client.get_tenant_info()
+    return tenant_info
```
Comment (adi88d) [inline Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py:390]:
```suggestion
    readable_output = tableToMarkdown(
        'Tenant Information', tenant_info, headerTransform=pascalToSpace, removeNull=True, is_auto_json_transform=True
    )
    return CommandResults(
        readable_output=readable_output,
        outputs_prefix=f'{INTEGRATION_CONTEXT_BRAND}.TenantInformation',
        outputs=tenant_info,
        raw_response=tenant_info
    )
```

**Review by adi88d — APPROVED** [review]
Great!

**Comment by adi88d** [comment]
@yilmazdincer i'm sorry in advance, we have some internal issues with the contribution process so your pull request release will be delay

**Pull request merged**